client/shipping-label/state/reducer.js

```diff
@@ -60,6 +60,8 @@ const generateUniqueBoxId = ( keyBase, boxIds ) => {
 	}
 };
 
+const toFixed = ( n ) => +n.toFixed( 6 );
```

Comment:
Ah, yeah, I'd forgotten about that. I'm OK with both too, but also prefer `_.round`